src/PackageCompilerX.jl

```diff
@@ -17,8 +17,22 @@ current_process_sysimage_path() = unsafe_string(Base.JLOptions().image_file)
 all_stdlibs() = readdir(Sys.STDLIB)
 
 yesno(b::Bool) = b ? "yes" : "no"
-bitflag() = Int == Int32 ? `-m32` : `-m64`
-march() = (Int == Int32 ? `-march=pentium4` : ``)
+
+function bitflag()
+    if Sys.ARCH == :aarch64
```

Comment:
The out put of `Sys.ARCH` is `:arm` on Raspberry Pi Series.
Therefore `if Sys.ARCH == :aarch64 || Sys.ARCH == :arm` is much appreciated if you consider 32bit system

c.f. Here is a example on my Raspberry Pi 4

```console
lsb_release  -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
pi@rpi4:~ $ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.3.1 (2019-12-30)
 _/ |\__'_|_|_|\__'_|  |
|__/                   |

julia> versioninfo()
Julia Version 1.3.1
Commit 2d5741174c (2019-12-30 21:36 UTC)
Platform Info:
  OS: Linux (arm-linux-gnueabihf)
  CPU: ARMv7 Processor rev 3 (v7l)
  WORD_SIZE: 32
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, cortex-a72)


julia> Sys.ARCH
:arm

julia> Sys.ARCH == :aarch64
false

```